cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -1535,3 +1540,51 @@ export function redrawShapeAsync(): ThunkAction {
         }
     };
 }
+
+export function trackerSettings(name: string, value: string | number): AnyAction {
+    return {
+        type: AnnotationActionTypes.TRACKER_SETTINGS,
+        payload: {
+            name,
+            value,
+        },
+    };
+}
+
+export function resetTrackerSettings(): AnyAction {
+    return {
+        type: AnnotationActionTypes.RESET_TRACKER_SETTINGS,
+    };
+}
+
+export function doTracking(body: any, taskId: string): ThunkAction {
+    return async (dispatch: ActionCreator<Dispatch>): Promise<void> => {
+        dispatch({
+            type: AnnotationActionTypes.DO_TRACKING,
+        });
+        const baseUrl = cvat.config.backendAPI.slice(0, -7);
+        try {
```

Comment:
``Try/catch`` construction is useless here because you use ``.then().catch()`` construction.
Let's use ``await`` instead, because this function is ``async``. It makes code easier.

Reply:
True. Have fixed it.